### DISCUSSION TITLE
Data Grid sort by date?

### DISCUSSION THREAD

**Discussion by azhengtwilio** [twilio-labs/paste#2488] [Help: Figma] [answered]
Hi, 
We're trying to implement a table that could be sorted by date, was reading up on the [data grid](https://paste.twilio.design/components/data-grid) paste component, seems like there is sorting for alphabetic order but wondering if we could do it with dates and time too?

![Screen Shot 2022-06-22 at 3 18 29 PM](https://user-images.githubusercontent.com/83669618/175162567-fdf90e93-8402-4f05-92d5-3ff53730681c.png)

**Comment by TheSisb** [comment, marked as answer]
Hi @azhengtwilio,

Here's a codesandbox showing how you'd do it: https://codesandbox.io/s/sortable-datetime-columns-datagrid-0wvcnm

In general, dates are sortable in JS you just need to make sure the data is an actually `Date` object. If you look at the `constants.tsx` file you will see our data is using Date objects.  During the render, I use `toString()` to display the string version, but you can use any [native Date functions ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) to display it however you like.
